airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -480,8 +480,8 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
             Metafields(config),
             CustomCollections(config),
             Collects(config),
-            OrderRefunds(config),
-            OrderRisks(config),
+            OrdersRefund(config),
```

Comment:
@bazarnov why are we renaming these streams? this would cause a backwards breaking change for customers. Is it currently breaking and that's why we are renaming? in this case the rename is safe (since it's currently breaking)

Reply:
This is reverted. No need to do this.